### DISCUSSION TITLE
Apply transform

### DISCUSSION THREAD

**Discussion by ccLiiFE** [CoBrALab/optimized_antsMultivariateTemplateConstruction#105] [General]
Hi!
I would like to thank you for this wonderful program.
I have been able to create a mean template of my 30 MRI T1 (acquired in 7T). 
I am getting a very nice  general template in final/average under the name template_sharpen_shapeupdate.nii.gz
Now I have sub cortical manually segmented areas (for each of my original T1) and I would like to apply the Transformation on these segmentation in order to see the overlaps.
My idea (maybe naive sorry) was to use antsApplyTransforms with the option 
-r  template_sharpen_shapeupdate.nii.gz
-n NearestNeighbor (for my segmentation) or BSpline (for my test on T1)
-o output

I tried it on the original T1 hoping to get the T1 that I have in final/resample.
Unfortunately, I don't obtain the same volume. Did I forget something? or misunderstood something?
Thanks a lot for your help!

**Comment by gdevenyi** [comment]
Hi, to resample data aligned to the original input images into the final space, see `commonspace_resample.sh`

**Comment by ccLiiFE** [comment]
I am really sorry I did not see it! 
I  saw the subjectspace_resample but did not the commonspace.
Anyway, this is fantastic!!! thanks a lot for your help!
